hikari/impl/interaction_server.py

```diff
@@ -206,25 +203,23 @@ def is_alive(self) -> bool:
         return self._server is not None
 
     async def _fetch_public_key(self) -> signing.VerifyKey:
-        # This is kept inline as pynacl is an optional dependency.
-        from nacl import signing
-
         if self._application_fetch_lock is None:
             self._application_fetch_lock = asyncio.Lock()
 
         application: typing.Union[applications.Application, applications.AuthorizationApplication]
         async with self._application_fetch_lock:
-            if self._verify:
-                return self._verify
+            if self._public_key:
+                return self._public_key
 
             if self._rest_client.token_type == applications.TokenType.BOT:
                 application = await self._rest_client.fetch_application()
 
             else:
                 application = (await self._rest_client.fetch_authorization()).application
 
-            self._verify = signing.VerifyKey(application.public_key)
-            return self._verify
+            print(application.public_key)
```

Comment:
Print left here

Reply:
That print isn't there any more, you reviewed an old commit